nixos/modules/misc/ids.nix

```diff
@@ -340,6 +340,7 @@
       cockroachdb = 313;
       zoneminder = 314;
       paperless = 315;
+      roon-server = 316;
```

Comment:
Ah, no need for that though. NixOS will automatically assign an {u,g}id if you don't set one.

Reply:
(this is what https://github.com/NixOS/nixpkgs/pull/65698 is aiming for)

Reply:
I'm curious: what scenario would specifically require a fixed predefined UID/GID for everyone if nix can just assign one? o:

Reply:
I can only think of some kind of service that needs to interact with other instances on different machines by means of moving files between them, and for some reason not having implemented uid adjustments. IIRC an old version of NFS might have been such a case.